### ISSUE TITLE
Complete jest test snippet

### ISSUE THREAD

**Issue by ahoseinian** [bryceosterhaus/atom-jest-snippets#2]
I was wondering if you like the following snippet to be added to this repo and if I can make a PR for that?

```
  'jtest':
    description: 'jest test'
    prefix: 'jtest'
    body: """
      import React from 'react'
      import {shallow} from 'enzyme'
      import $1 from './$1'

      describe('$1', () => {
        it('$2', () => {
          const wrapper = shallow(<$1 />)
          
        })
      })

    """
```
we can change the prefix as you wish

**Comment by bryceosterhaus** [comment]
How about leaving out the enzyme and react imports since those are react specific, and not mandatory for jest.

I am thinking something like
```
  'jsetup':
    description: 'Sets up jest test file'
    prefix: 'jsetup'
    body: """
      import $1 from './$1'

      describe('$1', () => {
        test('$2', () => {
            $3
        })
      })

    """
```

I would prefer this package to be as generic as possible, usable for any jest tests, not just for jest within a specific framework. If that sounds good to you, go ahead and send a pr.

**Comment by ahoseinian** [comment]
yeah that seems perfect given that you want to keep this repo generic to jest. I will make a PR asap.

**Comment by bryceosterhaus** [comment]
merged pr and released [v1.0.2](https://github.com/bryceosterhaus/atom-jest-snippets/releases/tag/v1.0.2)
